itests/standalone/extra/src/main/resources/classloading/exported-path-patterns.txt

```diff
@@ -26,6 +26,7 @@
 	^org/apache/xmlbeans/impl/schema
 	^org/apache/xmlbeans/impl/values
 	^org/apache/zookeeper(.*)
+    ^com/braintreegateway(.*)
```

Comment:
Fixed. 
Funny thing is that I cared about ordering but it looks like my brain automatically removes the internet suffix